src/Compilers/Core/Portable/RuleSet/RuleSetInclude.cs

```diff
@@ -80,6 +80,11 @@ private string GetIncludePath(RuleSet parent)
             List<string> found = new List<string>();
             string expandedPath = PortableShim.Environment.ExpandEnvironmentVariables(_includePath);
 
+            if (PathUtilities.IsUnixLikePlatform)
+            {
+                expandedPath = expandedPath.Replace('\\', PortableShim.Path.DirectorySeparatorChar);
```

Comment:
I'm assuming that the `\\` are introduced by `ExpandEnvironmentVariables` above. Why does it introduce those, rather than using the platform-specific characters in the first place?